Source/WebKit/WebProcess/InjectedBundle/DOM/InjectedBundleNodeHandle.cpp

```diff
@@ -228,7 +233,7 @@ RefPtr<WebImage> InjectedBundleNodeHandle::renderedImage(SnapshotOptions options
     }
 
     frameView->setNodeToDraw(m_node.get());
-    auto image = imageForRect(frameView, paintingRect, bitmapWidth, options);
+    auto image = imageForRect(frameView.get(), paintingRect, bitmapWidth, options);
```

Comment:
This doesn't pertain to making the code safer (or the rest of your patch), but I believe we prefer to not hide smart pointers behind `auto` where possible. Maybe we can do:

`RefPtr image = imageForRect(frameView.get(), paintingRect, bitmapWidth, options);`

as a drive-by fix here since `imageForRect()` returns a `RefPtr`.